.rubocop.yml

```diff
@@ -13,7 +13,25 @@ Metrics/BlockLength:
 ModuleLength:
   Enabled: false
 
-Style/EmptyMethod:
+Style/GlobalVars:
+  Enabled: false
```

Comment:
This one occurs only twice and only inside `spec/data/`. Either create a separate .rubocop.yml or suppress the rule locally.